lib/main.dart

```diff
@@ -83,7 +84,11 @@ main() async {
   // Setup services
   setup();
 
-  await web3WalletService!.init();
+  retry(
+    () => web3WalletService!.init(),
+    retryIf: (e) => e is SocketException,
```

Comment:
Should this also retry in the case of a TimeoutException, like in the package's [example](https://pub.dev/packages/retry)?

Reply:
That's probably a good idea. As far as I can see the WC documentation doesn't mention which exceptions are expected for the `init` method. I'll add the TimeoutException.